src/Interfaces.ts

```diff
@@ -1,8 +1,8 @@
-type IGrid = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+type GridInterface = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
```

Comment:
Maybe call it `GridType` while were at it, since it isn't actually an interface.